pkg/sql/show_fingerprints_test.go

```diff
@@ -140,6 +141,11 @@ func TestShowTenantFingerprintsProtectsTimestamp(t *testing.T) {
 	defer leaktest.AfterTest(t)()
 	defer log.Scope(t).Close(t)
 
+	// Under deadlock there isn't enough time for the CREATE TENANT txn to commit
+	// due to the intervals we lower to speed up this test. There is no difference
+	// otherwise when running this test under deadlock
+	skip.UnderDeadlockWithIssue(t, 121445, "Deadlock makes txns take too long to commit")
```

Comment:
nit: we use the withIssue tag to track an open issue that will be closed when we remove the skip line. Since this pr just closes  121445, no need to tag it.